### ISSUE TITLE
Video from integraged camera is not playing / shown

### ISSUE THREAD

**Issue by promosjotte** [electron/electron#12010]
* Electron version:
1.8.2
* Operating system:
Windows 10

### Expected behavior
Video from webcam should be shown.

### Actual behavior
I want to see the video stream from the integrated camera.
The Camera-Light is green (so camera is activated), but I see nothing on the video-tag.
The stream seems to keep pending...

`var that = this;
navigator.mediaDevices.getUserMedia({
	video : true
}).then(function(stream) 
{
	that._stream = stream;
	var eVideo = document.querySelector("video");
	eVideo.src = window.URL.createObjectURL(stream); 
	eVideo.play();	
}).catch(function() 
{
	console.log("could not connect to camera");
});`

![image](https://user-images.githubusercontent.com/14180294/36536642-05f2f998-17ce-11e8-8bc1-d8fb9ea4461e.png)

![image](https://user-images.githubusercontent.com/14180294/36536617-e88fa090-17cd-11e8-87bf-045fa7675bbc.png)

### How to reproduce

**Comment by sofianguy** [comment]
Thank you for taking the time to report this issue and helping to make Electron better.

The version of Electron you reported this on has been superseded by newer releases. 

If you're still experiencing this issue in Electron v2.0.x or later, please add a comment specifying the version you're testing with and any other new information that a maintainer trying to reproduce the issue should know.

I'm setting the `blocked/need-info` label for the above reasons.

Thanks in advance! Your help is appreciated.

**Comment by sofianguy** [comment]
Thank you for your issue!

We haven't gotten a response to our questions in our comment above. With only the information that is currently in the issue, we don't have enough information to take action. I'm going to close this but don't hesitate to reach out if you have or find the answers we need, we'll be happy to reopen the issue.

**Comment by promosjotte** [comment]
Hi @sofianguy,
thanks for your response and sorry about the response time. It is an old ticket and I figured out some things. The Camera is generally working when the App is not packaged (means when I start it from VS Code debug tool). The Camera can NOT be used when any packager is used. I can copy over the original electron.exe from downloaded Release (electron-v3.0.14-win32-x64.zip) and it works. 

This is still a problem in electron 3.xx and 4.xx.

**Comment by promosjotte** [comment]
still a problem with 6.9.0 ...

**Comment by mduriancik** [comment]
still a problem with 7.0.0 ...

**Comment by electron-triage** [comment]
Thank you for taking the time to report this issue and helping to make Electron better.

The version of Electron you reported this on has been superseded by newer releases. 

If you're still experiencing this issue in Electron 6.x.y or later, please add a comment specifying the version you're testing with and any other new information that a maintainer trying to reproduce the issue should know.

I'm setting the `blocked/need-info` label for the above reasons. This issue will be closed 7 days from now if there is no response.

Thanks in advance! Your help is appreciated.

**Comment by electron-triage** [comment]
Thank you for your issue!

We haven't gotten a response to our questions in our comment above. With only the information that is currently in the issue, we don't have enough information to take action. I'm going to close this but don't hesitate to reach out if you have or find the answers we need, we'll be happy to reopen the issue.

**Comment by robinwassen** [comment]
I bumped into the same effect (camera working on macOS but not Windows, Electron 5.x).

My problem was when I specified a fixed width and height it did not play well on Windows, but it worked in macOS on the same machine.

When I changed to **video: true** instead of specifying resolution settings it started to work.

Probably not related to the original poster since he also specified video: true rather than having specific resolutions.
